_includes/mobile/rewarded-server-side-configuration.md

```diff
@@ -0,0 +1,68 @@
+
+{: .table .table-bordered .table-striped }
+
+| Attribute | Type | Description | Example |  
+|-----------|------|-------------|---------|
+| `reward`             | object <br> (optional)   | Metadata provided by the publisher to describe the reward.                                                             |<code>{<br>&nbsp;"type": "SuperDollars",<br>&nbsp;"count": 10<br>}</code> | 
+| `reward.type`        | string   | Type of the reward in the app's coins. | `"SuperDollars"` | 
+| `reward.count`       | integer  | Amount of coins. | `10` | 
+| `reward.ext`         | object   | For future extensions. | <code>{<br>&nbsp;"ext": {}<br>}</code> | 
+| `completion`         | object <br> (optional)   | Describes the condition when the SDK should send a signal to the app that the user has earned the reward. |<code>{<br>&nbsp;"video": {<br>&nbsp;&nbsp;"endcard": {<br>&nbsp;&nbsp;&nbsp;"time": 5 <br>&nbsp;&nbsp;&nbsp;} <br>&nbsp;&nbsp;} <br>}</code> | 
+| `completion.banner`  | object   | Details for banner ad completion. |<code>{<br>&nbsp;"time": 5,<br>&nbsp;"event": "custom_event_url" <br>}</code> | 
+| `completion.banner.time`   | integer  | Period of time the banner ad is on screen. | `5` | 
+| `completion.banner.event`  | string   | URL with custom schema sent by the creative to indicate that the user did earn a reward. | `"rwdd://userDidEarnReward"` | 
+| `completion.video`   | object   | Details for video ad completion. |<code>{<br>&nbsp;"endcard": {<br>&nbsp;&nbsp;"time": 5 <br>&nbsp;} <br>}</code> | 
+| `completion.video.time`    | integer  | Period of time the video ad is on screen. | `10` | 
+| `completion.video.playbackevent` | string   | The playback event stage in the video. | `"start"`, `"firstquartile"`, `"midpoint"`, `"thirdquartile"`, `"complete"` | 
+| `completion.video.endcard` | object   | Properties for the end card. |<code>{<br>&nbsp;"time": 5 <br>}</code> | 
+| `completion.video.endcard.time` | integer  | Period of time the end card is on screen. | `5` | 
+| `completion.video.endcard.event` | string   | URL with custom schema sent by the creative for end card. | `"rwdd://userDidEarnReward"` | 
+| `close`              | object <br> (optional)   | Describes the ad close behavior after the reward is earned. |<code>{<br>&nbsp;"postrewardtime": 3,<br>&nbsp;"action": "autoclose"<br>}</code> | 
+| `close.postrewardtime` | integer  | Time interval (seconds) after reward event when SDK should close interstitial. | `3` | 
+| `close.action`       | string   | Action SDK should make: `"autoclose"` (close interstitial) or `"closebutton"` (show close button) | `"autoclose"` | 
+
+
+An example of an impression-level stored request:
+
+```json
+{
+  "ext": {
+    "prebid": {
+      "storedauctionresponse": {
```

Comment:
First, please make this an impression-level block.

Second, please make it a storedrequest rather than a stored response.

I don't believe passthrough should be specified by both the ORTB and the storedrequest, right? That's not clear from this example.

Reply:
The example of impression-level stored requests is fixed. 

We believe that the best way to provide `rwdd` params to the SDK is to use stored requests. In this case publishers (or host companies) can change them online.